censowalletintegration/src/main/java/co/censo/walletintegration/CensoWalletIntegration.kt

```diff
@@ -8,12 +8,16 @@ import android.net.Uri
 class CensoWalletIntegration: ContentProvider() {
     val apiUrl = "https://api.censo.co"
     val apiVersion = "v1"
-    val linkScheme = "censo-import"
+    val linkScheme = "censo-main"
+    val linkHost = "import"
```

Comment:
The other SDKs don't let you configure the `linkHost` and I'm not sure there's really a valid use case for that -- I think it'd be better to just hard-code it in `connect()`

Reply:
Updated to make it hardcoded in connect()